### PR TITLE
Add bandit and liccheck

### PR DESCRIPTION
**Pull request by eganjs**
* Refactor naming conventions for repo

### CODE REVIEW

File: {{cookiecutter.project_repo}}/Makefile [hunk 1]
```diff
@@ -47,9 +47,14 @@ clean-test: ## remove test and coverage artifacts
...
 
-lint: ## check style with flake8 and black
+lint: static-analysis ## check style with flake8 and black
 	flake8 {{ cookiecutter.project_slug }} tests setup.py
 	black --check --diff {{ cookiecutter.project_slug }} tests setup.py
+	mypy --ignore-missing-imports {{ cookiecutter.project_slug }} tests setup.py
+
+static-analysis: ## run static analysis and check licences of dependencies
+	bandit -r {{ cookiecutter.project_slug }}
+	liccheck -s liccheck.ini -r requirements.txt
```
Comment (MarcDenman) [inline {{cookiecutter.project_repo}}/Makefile:57]: @elduddz Can you take a look please at bandit and liccheck please? The allowed licenses are in this PR in a file called `liccheck.ini`

File: {{cookiecutter.project_repo}}/liccheck.ini [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+        apache software
+        apache software license
+        apache-software
+        asl
+        asl 2
+        bsd
+        bsd license
+        bsd-3-clause
+        cc-by
+        gnu lgpl
```
Comment (elduddz) [inline {{cookiecutter.project_repo}}/liccheck.ini:17]: lesser is certainly better, but I would always suggest some level of documentation around all the licenses being used, that that often helps people to understand where the issues are.

Are there only 2 levels? could you have an review_authorised?

File: {{cookiecutter.project_repo}}/liccheck.ini [hunk 3]
```diff
@@ -0,0 +1,43 @@
+# Authorized and unauthorized licenses in LOWER CASE
+[Licenses]
+authorized_licenses:
```
Comment (elduddz) [inline {{cookiecutter.project_repo}}/liccheck.ini:3]: what happens if the list changes, does anyone who has derived from this have to update their own liccheck?

Comment (eganjs) [reply {{cookiecutter.project_repo}}/liccheck.ini:3]: Yes, this is a challenge yet to be solved, if you have an ideas I would appreciate it very much

File: {{cookiecutter.project_repo}}/liccheck.ini (hunk 2 above)
Comment (eganjs) [reply {{cookiecutter.project_repo}}/liccheck.ini:17]: The way to force it to be reviewed is to remove it from the approved list and add individual packages as part of the review process in the `[Authorized Packages]` sections

Comment (elduddz) [reply {{cookiecutter.project_repo}}/liccheck.ini:17]: I think so, does that feel ok. I'm not too precious of my views on which licences we could allow, but I'm always cautious around the GNU ones.

File: {{cookiecutter.project_repo}}/liccheck.ini (hunk 3 above)
Comment (elduddz) [reply {{cookiecutter.project_repo}}/liccheck.ini:3]: I think that is always the issue. Even if you used this as a template project it still is only a view of what is correct at that moment in time.

File: {{cookiecutter.project_repo}}/CONTRIBUTING.rst [hunk 4]
```diff
@@ -59,10 +59,10 @@ Get Started!
 
 Ready to contribute? Here's how to set up `{{ cookiecutter.project_slug }}` for local development.
 
-1. Fork the `{{ cookiecutter.project_slug }}` repo on GitHub.
+1. Fork the `{{ cookiecutter.project_repo }}` repo on GitHub.
```
Comment (elduddz) [inline {{cookiecutter.project_repo}}/CONTRIBUTING.rst:62]: Actually.. you are asking for people to fork aren't you.

That might be how you keep the liccheck up-to-date.

**Pull request merged**